kubernetes/spilo_kubernetes.yaml

```diff
@@ -132,47 +127,77 @@ spec:
 apiVersion: v1
 kind: Endpoints
 metadata:
-  name: &cluster_name zalandodemo01
+  name: *cluster_name*
   labels:
     application: spilo
-    spilo-cluster: *cluster_name
+    spilo-cluster: *cluster_name*
 subsets: []
 
+# Services
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: *cluster_name*-master
```

Comment:
The name of the master (leader) service MUST match with the cluster name. Label selector is redundant here, because Patroni controls the corresponding [Endpoint](https://github.com/zalando/spilo/pull/325/files#diff-1cc5f12e11c6450ce292458f7fb8426cR128) and the service works automatically.